libbeat/outputs/tidb_slow_query/client.go

```diff
@@ -0,0 +1,233 @@
+package tidb_slow_query
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/elastic/beats/v7/libbeat/logp"
+	"github.com/elastic/beats/v7/libbeat/outputs"
+	"github.com/elastic/beats/v7/libbeat/publisher"
+	"github.com/go-sql-driver/mysql"
+	lru "github.com/hashicorp/golang-lru"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
Duplicated import.

Reply:
It's a driver initialization import. See https://github.com/go-sql-driver/mysql#usage.

Reply:
You have already imported the mysql in previous imports, then the driver is already initialized I believe.